include/tiled/Tileset.hpp

```diff
@@ -258,7 +267,7 @@ int tson::Tileset::getFirstgid() const
  * @return
  */
 
-const fs::path &tson::Tileset::getImagePath() const { return m_image; }
```

Comment:
Could you - instead of changing the existing `getImagePath()` create a new `getFullImagePath()` that references your new variable? 
This is mostly not to introduce a breaking change. Besides: `getImagePath()` would return a relative path just as it's presented in `Tiled`, which should be an option. 

Other than that: I fully support your suggestion :slightly_smiling_face:

Reply:
Good point, let me change that quickly.

Reply:
Done, thanks for the feedback.